README.md

```diff
@@ -106,7 +106,7 @@ dht      smlbench
 Then do
 
 ```
-> TESTGROUND_BASEDIR=`pwd` testground run dht/lookup-peers --builder=docker:go
+> TESTGROUND_BASEDIR=`pwd` testground run dht/lookup-peers --builder=docker:go --runner=local:docker
```

Comment:
I had added the previous command before and I was able to run it. Today, I had to add the runner specifically. Unsure what changed